File: PHP/TestCases/AndroidWebTest.php
<?php
use PHPUnit\Framework\TestCase;
include('Utils/Driver.php');
include('Utils/DesiredCapabilities.php');
include('Objects/HerokuWeb.php');

class AndroidWebTest extends TestCase {
  public $cap, $driver_, $testPage, $successMsg;

  public function setUp() {
    $this->caps = GetAndroidWebDesiredCapability();
    $this->driver_ = CreateDriver($this->caps);
    $this->testPage = new HerokuWeb($this->driver_);
    $this->successMsg = "You logged into a secure area!";
  }

  public function test_login() {
    print "\n ***** should run test successfully with correct username and password *****";
    $this->assertContains($this->successMsg, $this->testPage->getLoginMessage());
  }

  public function tearDown() {
    QuitDriver($this->driver_);
  }
}
?>

File: Ruby/Testcases/AndroidWebTest.rb
require 'selenium-webdriver'
require 'test/unit'
require_relative '../Objects/HerokuWeb'
require_relative '../Utils/Driver'
require_relative '../Utils/DesiredCapabilities'

class AndroidWebTest < Test::Unit::TestCase
  def setup
    @cap = getAndroidWebDesiredCapability
    @driver = CreateDriver @cap
    @testPage = HerokuWeb.new @driver
    @loginMsg = 'You logged into a secure area'
    @failMsg = 'Login message is not matched'
  end

  def test_should_login_successfully
    @testPage.login
    assert_includes @testPage.getLoginMessage, @loginMsg, @failMsg
  end

  def teardown
    QuitDriver @driver
  end

end
